### PR TITLE
Cleanup imports and resolve success

### PR DESCRIPTION
**Pull request by dalder**

### CODE REVIEW

**Review by lenardpalko — APPROVED** [review]
There is a typecheck issue, but other than that looks good.

File: src/repositories/DocumentRepository.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 import { HttpClient } from '../utilities/httpClient';
 import type { DocumentApi } from '../types';
-import type { FrontifyDocument } from '../types/FrontifyDocument';
+import type { FrontifyDocument } from '../types';
```
Comment (SamuelAlev) [inline src/repositories/DocumentRepository.ts:5]: Can be merged with the line on the top

File: src/repositories/DocumentGroupRepository.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import type { DocumentGroupApi } from '../types/DocumentGroup';
-import type { FrontifyDocumentGroup } from '../types/FrontifyDocumentGroup';
+import type { DocumentGroupApi } from '../types';
+import type { FrontifyDocumentGroup } from '../types';
```
Comment (SamuelAlev) [inline src/repositories/DocumentGroupRepository.ts:4]: Both lines can be merged

File: src/repositories/ColorRepository.ts [hunk 3]
```diff
@@ -1,8 +1,8 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
 import { HttpClient } from '../utilities/httpClient';
-import { ColorPaletteApi } from '../types/ColorPalette';
-import { ColorApi, ColorApiCreate, ColorApiPatch } from '../types/Color';
+import { ColorPaletteApi } from '../types';
+import { ColorApi, ColorApiCreate, ColorApiPatch } from '../types';
```
Comment (SamuelAlev) [inline src/repositories/ColorRepository.ts:5]: Same and you can change it to « import type »

File: src/repositories/DocumentRepository.ts (hunk 1 above)
Comment (dalder) [reply src/repositories/DocumentRepository.ts]: done

File: src/repositories/DocumentGroupRepository.ts (hunk 2 above)
Comment (dalder) [reply src/repositories/DocumentGroupRepository.ts]: done

File: src/repositories/ColorRepository.ts (hunk 3 above)
Comment (dalder) [reply src/repositories/ColorRepository.ts]: done

File: src/repositories/ColorRepository.spec.ts [hunk 4]
```diff
@@ -11,7 +11,7 @@ import {
     updateColor,
 } from './ColorRepository';
 import { HttpClient } from '../utilities/httpClient';
-import { ColorApiCreateDummy, ColorApiDummy, ColorApiPatchDummy } from '../tests/ColorApiDummy';
+import { ColorApiCreateDummy, ColorApiDummy, ColorApiPatchDummy } from '../tests';
 import { FrontifyColorCreateDummy, FrontifyColorDummy, HttpUtilResponseDummy } from '../tests';
```
Comment (lenardpalko) [inline src/repositories/ColorRepository.spec.ts:15]: It seems that these two lines can be combined as well.

Comment (dalder) [reply src/repositories/ColorRepository.spec.ts]: done

**Review by lenardpalko — APPROVED** [review]

**Pull request merged**